### DISCUSSION TITLE
MacOS support

### DISCUSSION THREAD

**Discussion by Brog33** [bytespiller/sidplaywx#3] [General]
wxWidgets are also supported on MacOS. Would you be down to make a mac build too? ( I'd love to beta test )

**Comment by bytespiller** [comment]
After the proper Linux support is implemented and the build scripts improved, it should be easier to make a Mac build too. I'll see what I can do, but it probably won't be soon.

**Reply by Brog33** [reply]
That's cool. Hope the Linux support goes smoothly. I'll be testing that one too :)
